metaschema-cli/src/main/java/gov/nist/secauto/metaschema/cli/commands/metapath/MetapathCommand.java

```diff
@@ -11,6 +11,9 @@ public class MetapathCommand
     extends AbstractParentCommand {
   private static final String COMMAND = "metapath";
 
+  /**
+   * Construct a new Metapath command.
```

Comment:
Minor, non-blocking: wording to make it clear it's a constructor, but not something I am passionate about.

```suggestion
   * Constructor for a new Metapath command.
```